scripts/regression_test/extension_regression_test.yml

```diff
@@ -107,19 +107,21 @@ jobs:
     parameters:
       CLIExtensionRepoPath: ./azure-cli-extensions
   - task: AzureCLI@1
-    displayName: 'Rerun tests'
+    displayName: 'az login'
     inputs:
       azureSubscription: $(AZURE_SDK_TEST_SUB_CONNECTED_SERVICE)
       scriptLocation: inlineScript
       inlineScript: |
-        set -ev
-  
-        source env/bin/activate
-        cd azure-cli-extensions
-  
         az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
   
-        python ../scripts/ci/automation_full_test.py "12" "$(Instance_idx)" "latest" "" "True" "extension"
+  - bash: |
+      set -ev
+      
+      source env/bin/activate
+      cd azure-cli-extensions
+      python ../scripts/ci/automation_full_test.py "12" "$(Instance_idx)" "latest" "" "True" "extension"
+    displayName: 'Rerun tests'
```

Comment:
`azdev` can't find extension list when running in `AzureCLI@1`, move it to new job.

Reply:
What is the error and do we know why?

Reply:
I think it's better not to change it until we figure out why.